packages/@sfx/core/test/unit/common/utils/core.test.ts

```diff
@@ -0,0 +1,315 @@
+import { expect, spy } from '../../../utils';
```

Comment:
It looks like this file is not being run in the node testing environment as the `.mocharc.json` is not set up to watch subdirectories inside of the common/node directories.